### ISSUE TITLE
[WCAG 2.2 Draft Feedback] 3.2.7 Visible Controls (Level AA)

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1895]
"Success Criterion 3.2.7 Visible Controls (Level AA): Where receiving pointer hover or keyboard focus triggers user interface components to be visible, information needed to identify that user interface components are available is visible, except when:
The information needed to identify the user interface components is available through an equivalent component that is visible on the same page or on a different step in a multi-step process without requiring pointer hover or keyboard focus;
The component is provided specifically to enhance the experience for keyboard navigation;
A mechanism is available to make the information persistently visible;
Hiding the information needed to identify the component is essential."

Most of our comments/proposals are requests to: consider renaming the SC, provide more information about the approved exceptions in the test, use more explicit language in the "when the controls are needed" and "persistent visible entry point" references, explicitly address timeouts, and provide more information in Sufficient Techniques.   

Please find detailed specifics covered in our [3.2.7 Visible Controls (Level AA) Google Doc](https://docs.google.com/document/d/1yP-XVuQxULNjDA30edHZxiKREuyTf2BsJ6dxoLtwz5c/edit#).

**Comment by mbgower** [comment]
Draft response:
Both the normative and Understanding document for this Success Criterion have been substantially revised. For the latest version see https://raw.githack.com/w3c/wcag/main/understanding/22/visible-controls.html

Responses have also been made in the your [detailed comments ](https://docs.google.com/document/d/1yP-XVuQxULNjDA30edHZxiKREuyTf2BsJ6dxoLtwz5c/edit#)

Please review both the new version and the responses, and advise if any comments remain unaddressed.
In addition, the following have been done:
- a PR to address some of the requests for text improvement
- identification of some visuals that should be added to the Understanding document, providing examples of passing and failing visible controls
